### DISCUSSION TITLE
Demo example PHPSandbox

### DISCUSSION THREAD

**Discussion by AndreiTelteu** [dedoc/scramble#176] [Ideas]
Hi!

I made a demo notebook with laravel 10 and scramble on PHPSandbox.io

[![Test on PHPSandbox](https://phpsandbox.io/img/brand/badge.png)](https://phpsandbox.io/e/x/lcgz0?layout=Preview&iframeId=58glqjpk5m&theme=dark&defaultPath=%2F&showExplorer=no)

I could not get route query params working in this demo, but it's still early for this project.

I would be nice to have a notebook like this one available to play with when this projects launches.

**Comment by romalytvynenko** [comment]
@AndreiTelteu Hey!

When you say "when this projects launches", which project do you mean?

**Reply by AndreiTelteu** [reply]
@romalytvynenko scramble, the open source plugin, not the hosted version.

I saw this warning and I assumed this project is in some kind o beta stage

>   Warning Package is in early stage. It means there may be bugs

Also the website dedoc.co has an early access mailing list, it looks like a project not fully launched yet. Is that for the hosted version ?

**Reply by romalytvynenko** [reply]
Oh, I see.

Yeah, so early access mailing list is for hosted API docs.

Scramble is a docs generator for Laravel, and while it is in early stage, I'm considering it as "launched" :D 

Speaking of a playground – this is a great idea.

I'd really like to improve docs with a ton of examples (code on the left, resulting docs UI on the right), but also to have some sort of sandbox so people can play around.

**Reply by AndreiTelteu** [reply]
Cool. PHPSandbox is just right for this. 

https://phpsandbox.io/n/lcgz0#app/Http/Controllers/PostController.php

Click Share > Embed > Customize and you have a customizable iframe sandbox.
